app/scripts/modules/core/src/pipeline/config/triggers/artifacts/artifact.component.ts

```diff
@@ -70,9 +70,8 @@ class ArtifactCtrl implements IController {
     if (this.artifactAccounts) {
       options = options.filter(o => {
         const isCustomArtifact = o.customKind;
-        const isPublic = !!o.isPubliclyAccessible;
         const hasCredential = this.artifactAccounts.find(a => a.types.includes(o.type));
-        return isCustomArtifact || isPublic || hasCredential;
```

Comment:
I'm a bit confused about the scope of changes.  If we're flagging the new artifact experience why do removals like this remain?

Reply:
because with https://github.com/spinnaker/clouddriver/pull/3424, isPublic is not needed.